xtask/coverage/src/symbols/msts.rs

```diff
@@ -222,12 +232,7 @@ struct SymbolsFile {
 fn parse_symbol(input: &str) -> Option<Symbol> {
     let (input, _) = parse_str(input, ">")?;
     let (input, name) = parse_until_chr(input, |x| x.is_whitespace() || x == ':')?;
-    if name.contains('.')
-        || name.contains('[')
-        || name.contains('\"')
-        || name.contains('\'')
-        || name == "undefined"
```

Comment:
Why was `"undefined"` removed?

Reply:
Because it blocked some success on the coverage. This seems to be a legacy check.